cmd/webhooks.go

```diff
@@ -1,16 +1,205 @@
+			for _, value := range TriggerTypes {
+				if strings.Contains(value, toComplete) {
+					candidates = append(candidates, value)
+				}
+			}
+			// if there are no valid suggestions, suggest all available options
+			if len(candidates) == 0 {
+				candidates = TriggerTypes
+			}
+			return candidates, cobra.ShellCompDirectiveNoFileComp
+		}
+		return nil, cobra.ShellCompDirectiveError
+	},
+	PreRunE: func(cmd *cobra.Command, args []string) error {
+		triggerType := args[1]
+		for _, b := range TriggerTypes {
+			if b == triggerType {
+				return nil
+			}
+		}
+		return fmt.Errorf("unknown Trigger Type: '%s'.\ntrigger_type must be one of: %+q\n", triggerType, TriggerTypes)
+	},
+	Run: func(cmd *cobra.Command, args []string) {
+		siteId := args[0]
+		triggerType := args[1]
+		url := args[2]
+
+		c := internal.NewClient(ApiToken)
+
+		payload := strings.NewReader(fmt.Sprintf(`{
+        "triggerType": "%s",
+        "url": "%s"
+    }`, triggerType, url))
+
+		body, err := c.Post([]string{"sites", siteId, "webhooks"}, payload)
+		if err != nil {
+			log.Fatalf("Request failed: %v", err)
+		}
+
+		var response CreateWebhooksResponse
+		err = json.Unmarshal(body, &response)
+		if err != nil {
+			log.Fatalf("Failed to unmarshal response body: %v", err)
+		}
+		fmt.Printf("%s\n", response.Id)
+	},
+}
+
+// ListWebhooksResponse represents a response to the list request in Webflow.
+// See: https://developers.webflow.com/reference/list-webhooks.
+type ListWebhooksResponse []Webhook
+
+// listWebhooksCmd represents the command to list webhooks for a site in Webflow.
+var listWebhooksCmd = &cobra.Command{
+	Use:   "list [site_id]",
+	Short: "list webhooks for a site",
+	Args:  cobra.ExactArgs(1),
+	Run: func(cmd *cobra.Command, args []string) {
+		siteId := args[0]
+
+		c := internal.NewClient(ApiToken)
+		body, err := c.Get([]string{"sites", siteId, "webhooks"})
+		if err != nil {
+			log.Fatalf("Request failed: %v", err)
+		}
+
+		var response ListWebhooksResponse
+		err = json.Unmarshal(body, &response)
+		if err != nil {
+			log.Fatalf("Failed to unmarshal response body")
+		}
+		for _, webhook := range response {
+			fmt.Printf("%s\t%s\t%s\n", webhook.Id, webhook.TriggerType, webhook.Url)
+		}
+	},
+}
+
+// DeleteWebhooksResponse represents a response to the remove webhook request in Webflow.
+// See: https://developers.webflow.com/reference/removewebhook.
+type DeleteWebhooksResponse struct {
+	Deleted int
+}
+
+// deleteWebhooksCmd represents the command to remove a webhook for a site in Webflow.
+var deleteWebhooksCmd = &cobra.Command{
+	Use:   "delete [site_id] [webhook_id]",
+	Short: "delete a webhook for a site",
+	Args:  cobra.ExactArgs(2),
+	Run: func(cmd *cobra.Command, args []string) {
+		siteId := args[0]
+		webhookId := args[1]
+		c := internal.NewClient(ApiToken)
+
+		_, err := c.Delete([]string{"sites", siteId, "webhooks", webhookId})
+		if err != nil {
+			log.Fatalf("Unable to delete webhook: %v", err)
+		}
```

Comment:
If deleting a webhook is successful will there be any feedback for the user?

Reply:
In unix philosophy, we avoid unnecessary noise so there is little value in printing "success" when the exit code already communicates that.